src/admin-rpc/init.ts

```diff
@@ -125,6 +128,76 @@ export const AdminApi: FastifyPluginCallback<Record<never, never>, Server, TypeB
     }
   );
 
+  fastify.post(
+    '/import-contract',
+    {
+      schema: {
+        description:
+          'Imports a smart contract from the Stacks API and refreshes its token metadata',
+        body: Type.Object({
+          contractId: Type.RegEx(SmartContractRegEx),
+          tokenIds: Type.Optional(Type.Array(Type.Integer())),
```

Comment:
Sorry if I missed it, but what is tokenIds used for?

Reply:
Oops, good catch, this was copy pasta